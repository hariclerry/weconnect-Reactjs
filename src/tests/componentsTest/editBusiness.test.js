import React from "react";
import { shallow, mount } from "enzyme";
import { MemoryRouter } from "react-router-dom";
import jwt from "jsonwebtoken";
import weconnectStore from "store";
import EditBusinessContainer from "containers/business/edit";

const loginUserMock = {
  access_token: jwt.sign(
    { email: "clerry@gmail.com", password: "this199" },
    "hard to guess string"
  )
};

describe("EditBusiness component", () => {
  beforeEach(() => {
    localStorage.setItem("token", loginUserMock.access_token);
  });
  it("should match snapshot", () => {
    const wrapper = shallow(<EditBusinessContainer />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should render the EditBusiness component", () => {
    const mountComponent = shallow(
      <MemoryRouter>
        <EditBusinessContainer store={weconnectStore} />
      </MemoryRouter>
    );
    expect(mountComponent.length).toBe(1);
  });
});
