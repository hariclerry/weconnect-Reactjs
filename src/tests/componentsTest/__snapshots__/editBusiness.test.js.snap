// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditBusiness component should match snapshot 1`] = `<Route />`;
